site/content/ytt/docs/develop/how-to-export-schema.md

```diff
@@ -172,6 +182,16 @@ Indicates whether `null` is also allowed.
 - when a data value is annotated `@schema/nullable` or `@schema/type any=True`, this property is included and set to `true`;
 - otherwise, this property is omitted.
 
+### `deprecated`
+
+(As of v0.39.0+)
+
+Marks a data value as targeted for removal while still remaining defined and used.
```

Comment:
```suggestion
Indicates if a key is deprecated.
```

It feels like the description of `deprecated` should match closer to `additionalProperties ` and `nullable`, since their values are all bool.